float32 y_dir
float32 x_dir
float32 dist
---
bool status